.github/workflows/integration.yaml

```diff
@@ -168,15 +134,12 @@ jobs:
         with:
           fetch-depth: 0
 
-      #TODO: there may be a more idiomatic way to do this
-      - name: Check if we should skip this build
-        id: job-canceller
-        run: echo "::set-output name=cancelled::${{ matrix.merge == 'merged' && github.event_name != 'pull_request'}}"
-
-      # merge with master(or whatever target branch) so we are actually testing what will happen after PR merges, not just this branch
-      - name: Merge target branch
-        run: git merge origin/${{ github.event.pull_request.base.ref }}
-        if: ${{ matrix.merge == 'merged' && github.event_name == 'pull_request' }}
+      - name: Setup Environment
```

Comment:
Maybe it is the diff being weird, but didn't we do this up above, and isnt this the end of the job now?